js_modules/dagit/packages/core/src/runs/RunsRootNew.tsx

```diff
@@ -192,29 +192,27 @@ export const RunsRoot = () => {
 
             return (
               <>
+                <Box flex={{direction: 'column', gap: 8}}>
+                  {currentTab !== 'all' ? (
+                    <Box flex={{direction: 'row', gap: 8}}>
+                      {filterTokens
+                        .filter((token) => token.token === 'status')
+                        .map(({token, value}) => (
+                          <Tag key={`${token}:${value}`}>{`${token}:${value}`}</Tag>
+                        ))}
+                    </Box>
+                  ) : null}
+                  <RunsFilterInput
```

Comment:
I moved this out so that both the `renderError` and this path have the same component structure:
```
<Box>
  <RunsFilterInput>
```
which means React will be able to keep the component's state when swapping between loading and not loading.